spec/lib/tufts/workflow_setup_spec.rb

```diff
@@ -28,8 +28,6 @@
     )
     role_names = roles.map { |r| Sipity::Role.find(r.role_id).name }
 
-    expect(role_names).to contain_exactly(
-      "publishing"
-    )
+    expect(role_names.include?("publishing")).to eq true
```

Comment:
:+1: there is also an `include` matcher we could use `expect(role_names).to include('publishing')`. That is equivalent to this, and I'm happy to merge as is when it's :green_heart:.